packages/demo-app/src/include.d/vite-end.d.ts

```diff
@@ -0,0 +1,2 @@
+// eslint-disable-next-line import/no-unassigned-import
+import 'vite/client';
```

Comment:
A little bit unexpected, since this is not included in #6314. What does this do?

Reply:
<img width="510" alt="image" src="https://github.com/user-attachments/assets/3718c3be-59cd-451a-868c-33a7cdcc8b3d">

replace some homemade type definitions

Reply:
Aha, nice